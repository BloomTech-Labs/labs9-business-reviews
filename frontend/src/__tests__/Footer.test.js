import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import FooterComponent from '../components/Footer';

describe('<FooterComponent />', () => {
  it('renders', () => {
    shallow(<FooterComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<FooterComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
  it('should contain 1 span', () => {
    let app = shallow(<FooterComponent />);
    expect(app.find("span").length).toBe(1);
  });
  it('should contain 1 div', () => {
    let app = shallow(<FooterComponent />);
    expect(app.find("div").length).toBe(1);
  });
  it('should contain 1 img', () => {
    let app = shallow(<FooterComponent />);
    expect(app.find("img").length).toBe(1);
  });
  it('should contain 1 "a" tag with text "Meet the Team"', () => {
    let app = shallow(<FooterComponent />);
    expect(app.find("a").length).toBe(1);
    expect(app.find("a").text()).toMatch("Meet the Team");
  });
});
