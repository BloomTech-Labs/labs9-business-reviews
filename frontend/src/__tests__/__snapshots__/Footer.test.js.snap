// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterComponent /> matches snapshot 1`] = `
<styled.div>
  <span>
    Â© 2019 Bonafind
  </span>
  <div
    className="meetTheTeam"
  >
    <img
      alt=""
      className="github"
      src="github.svg"
    />
    <a
      href="https://github.com/Lambda-School-Labs/labs9-business-reviews"
    >
      Meet the Team
    </a>
  </div>
</styled.div>
`;
